### ISSUE TITLE
CriticalCSS

### ISSUE THREAD

**Issue by RobbieTheWagner** [addyosmani/ember-progressive-webapp#9]
Hi @addyosmani, I was wondering what your recommendations were for running criticalCSS with Ember were? I see you reference a file as your critical-css, but not sure the best way to generate it?

**Comment by RobbieTheWagner** [comment]
@addyosmani do you have any recommendations for this? I would love to work on implementing critical for Ember, but probably need to do some magic stuff with the build to get it working.

**Comment by ianpetzer** [comment]
@rwwagner90 I'm also interested in using criticalCSS with Ember. It seems a bit daunting though!

I've been looking at this app and it doesn't appear to be generating _critical_ css, rather it is just inlining the content of the whole apps css as show [here](https://github.com/addyosmani/ember-progressive-webapp/blob/a4f346d8d292256eb6108db33862c9d54d62c914/ember-cli-build.js#L25)

**Comment by RobbieTheWagner** [comment]
@ianpetzer yeah, we would need to make some sort of ember addon to determine the critical CSS and inline it. Would be a lot of work. Inlining all CSS is not an option for large apps. Makes things very slow with a huge initial payload.
